explorer-api-common/src/main/java/org/zowe/api/common/security/SpringSecurityConfig.java

```diff
@@ -5,44 +5,27 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Copyright IBM Corporation 2018, 2018
+ * Copyright IBM Corporation 2018, 2020
  */
 package org.zowe.api.common.security;
 
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-import org.springframework.security.web.AuthenticationEntryPoint;
-import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
 
 @Configuration
 @EnableWebSecurity
 public class SpringSecurityConfig extends WebSecurityConfigurerAdapter {
 
-    @Autowired
-    CustomAuthenticationProvider authenticationProvider;
-
-    @Autowired
-    private AuthenticationEntryPoint authEntryPoint;
-
     private static final String[] AUTH_WHITELIST = {
             // -- swagger ui
             "/swagger-resources/**", "/swagger-ui.html", "/v2/api-docs", "/webjars/**"};
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         // TODO - re-enable csrf?
-        http.csrf().disable().authorizeRequests().antMatchers(AUTH_WHITELIST).permitAll().antMatchers("/api/**/*")
-                .authenticated().and().httpBasic().authenticationEntryPoint(authEntryPoint).and()
-                .logout().logoutRequestMatcher(new AntPathRequestMatcher("/api/v1/**/logout"));
```

Comment:
I guess that the logout mechanism is by just logging out from the gateway now?

Reply:
APIML squad is working on a proper logout endpoint that will invalidate the JWT, which will be the logout mechanism. Once that is delivered we should also front that with an api in each of data-sets and jobs for completeness. but a user could just use the endpoint directly with the gateway